sdk/version.go

```diff
@@ -0,0 +1,3 @@
+package sdk
```

Comment:
After speaking with Pavlos, we decided to track this manually for now. This is used in logs and will be used in metrics and so on to track which versions of which components are running.